openg2p_disbursement/models/openg2p_disbursement_batch_transaction.py

```diff
@@ -251,16 +251,21 @@ def create_bulk_transfer(self):
     def get_auth_token(self):
         try:
             headers = {
-                'Platform-TenantId': os.environ.get("tenantName"),
-                'Authorization': 'Basic Y2xpZW50Og==',
-                'Content-Type': 'text/plain',
+                "Platform-TenantId": os.environ.get("tenantName"),
+                "Authorization": "Basic Y2xpZW50Og==",
+                "Content-Type": "text/plain",
             }
             params = {
-                'username': os.environ.get("username"),
-                'password': os.environ.get("password"),
-                'grant_type': os.environ.get("grant_type"),
+                "username": os.environ.get("username"),
+                "password": os.environ.get("password"),
+                "grant_type": os.environ.get("grant_type"),
             }
-            response = requests.post('http://ops-bk.sandbox.fynarfin.io/oauth/token', params=params, headers=headers,verify=False)
+            response = requests.post(
+                "http://ops-bk.sandbox.fynarfin.io/oauth/token",
```

Comment:
this URL should not be added to the code, this should be removed from the code and, should be added as a config or environment variable :  eg: `os.environ.get("ops-url"),`

Reply:
Will make it to add it from environment variable.